### PR TITLE
Replace with @ember-decorators/utils

### PR DESCRIPTION
**Pull request by snewcomer**
https://github.com/ember-decorators/utils

### CODE REVIEW

File: addon/index.js [hunk 1]
```diff
@@ -1,54 +1,14 @@
...
-
-function handleDescriptor(callback, target, key, desc, params = []) {
-  return {
-    enumerable: desc.enumerable,
-    configurable: desc.configurable,
-    writeable: desc.writeable,
-    initializer: function() {
-      if (!desc.writable) {
-        throw new Error('ember-concurrency-decorators does not support using getters and setters');
-      }
```
Comment (snewcomer) [inline addon/index.js]: ah dang...forgot about this.  probably need to make sure this is still in there....

**Comment by snewcomer** [comment]
@machty just pinging this PR :taco: no rush though!

**Comment by machty** [comment]
Thanks for the contribution, and the ping! LGTM

**Comment by machty** [comment]
pushed e-c-d 0.0.3

**Pull request merged**